src/index.html

```diff
@@ -4,19 +4,107 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Moyo header</title>
+    <title>MOYO Header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
+  <link
+    rel="preconnect"
+    href="https://fonts.googleapis.com"
+  />
+  <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+  />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+    rel="stylesheet"
+  />
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+      <nav>
+        <ul class="menu-list">
+          <li class="list-item">
+            <a
+              href="#"
+              class="item-link is-active"
```

Comment:
Fix the classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
```